install/installer/pkg/common/common.go

```diff
@@ -337,6 +337,20 @@ func NodeAffinity(orLabels ...string) *corev1.Affinity {
 	}
 }
 
+func Replicas(ctx *RenderContext, component string) *int32 {
```

Comment:
IMO, I feel this makes the replica config more implicit, than explicit. For it to work, you've gotta (somewhat opaquely) specify it in the `PodConfig[component]` config which can be harder to find. 

This will also become somewhat problematic when we actually want to use a pod-autoscaler as this config will no longer make sense as the AutoScaler has a lot more config.

I'm basing this on a similar system we had in the past and it turned out to be very beneficial to just be extremely explicit, without helpers. The power of a full programming language to render config is awesome, but also quite a challenge to ensure we do keep it simple.

Reply:
> IMO, I feel this makes the replica config more implicit, than explicit. For it to work, you've gotta (somewhat opaquely) specify it in the PodConfig[component] config which can be harder to find.

I generally agree on the implicit connection via `component string` part. Here it's helpful that because using the "experimental" section is a bit noisy atm. And it follows the existing pattern, so ok for me, especially as it allows us to move forward. :+1: